content/administration/maintain/online.rst

```diff
@@ -12,32 +12,48 @@ To manage a database, sign in to https://www.odoo.com and access the `database m
 .. note::
    Make sure you are connected as the administrator of the database you want to manage.
 
-.. image:: online/dropdown-menu.png
-   :align: right
-   :alt: Clicking on the gear icon opens the drop-down menu.
+A :guilabel:`gear icon` and an :guilabel:`upgrade button` (represented by an arrow pointing upwards)
+are available next to your database name.
```

Comment:
Not sure the "represented by an arrow pointing upwards) is enough....  maybe it could lead to confusion. Adding an image is better maybe?

Reply:
the image is right below. I didn't find this symbol to insert it in the text directly...